src/controllers/softtakeover.cpp

```diff
@@ -102,32 +102,74 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
 
     qint64 currentTime = Time::elapsedMsecs();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == -1) {
         ignore = true;
         // Change the stored time (but keep it far away from the current time)
         //  so this block doesn't run again.
-        m_time = 1;
-//         qDebug() << "SoftTakeover::ignore: ignoring the first value"
-//                  << newParameter;
-    } else if ((currentTime - m_time) > SUBSEQUENT_VALUE_OVERRIDE_TIME_MILLIS) {
```

Comment:
Moving this to the places below, will ignore some more values as before. 
* if same side changes above thresholds

I have never understood, why we need the timer at all. What is the use case for this?